### PR TITLE
Orchestration monolith with 1 service and rest request v. 1.0

### PR DESCRIPTION
**Pull request by romanvlad95**
Created basic orchestrator monolith with only one service

### CODE REVIEW

File: api_services/user_service.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    def __init__(self, host: str):
+        self.host: str = host
+
+    async def fetch_user_data(self, user_id: str) -> Dict[str, Optional[str]]:
+        try:
+            async with grpc.aio.insecure_channel(self.host) as channel:
+                stub = UserServiceStub(channel)
+                request = UserRequest(user_id=user_id)
+                response = await stub.GetUserDetails(request)
+                if response.user_id:
```
Comment (Slavkata) [inline api_services/user_service.py:16]: Use explicit if checks

File: services/services.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+class SubscriptionService(subscription_pb2_grpc.SubscriptionServiceServicer):
+    def __init__(self):
+        self.orchestrator = Orchestrator()
+
+    async def GetSubscriptionDetails(self, request: subscription_pb2.SubscriptionRequest, context: ServicerContext) -> subscription_pb2.SubscriptionResponse:
+        user_id = request.user_id
+        subscription_data = SUBSCRIPTIONS_DB.get(user_id)
+
+        if subscription_data:
```
Comment (Slavkata) [inline services/services.py:35]: Use explicit Ifs

File: services/services.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+class UserService(user_pb2_grpc.UserServiceServicer):
+    def __init__(self):
+        self.orchestrator = Orchestrator()
+
+    async def GetUserDetails(self, request: user_pb2.UserRequest, context: ServicerContext) -> user_pb2.UserResponse:
+        user_id = request.user_id
+        user_data = USER_DB.get(user_id)
+
+        if user_data:
```
Comment (Slavkata) [inline services/services.py:16]: Use explicit Ifs

File: services/api.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+from hypercorn.config import Config
+from orchestrator.orchestrator import Orchestrator
+
+app = Flask(__name__)
+
+@app.route('/orchestrate', methods=['POST'])
+async def orchestrate() -> Response:
+    data = request.json
+    user_id = data.get('user_id')
+    if not user_id:
```
Comment (Slavkata) [inline services/api.py:12]: Use explicit Ifs

**Review by Slavkata — CHANGES_REQUESTED** [review]

File: services/api.py (hunk 4 above)
Comment (romanvlad95) [reply services/api.py]: Done, all inexplicit ifs are rewritten

File: services/services.py (hunk 2 above)
Comment (romanvlad95) [reply services/services.py]: Done, all inexplicit ifs are rewritten

File: api_services/user_service.py (hunk 1 above)
Comment (romanvlad95) [reply api_services/user_service.py]: Done, all inexplicit ifs are rewritten

**Comment by romanvlad95** [comment]
All messages are resolved

**Review by Slavkata — APPROVED** [review]

**Review by Slavkata — DISMISSED** [review]

**Review by Slavkata — APPROVED** [review]

**Pull request merged**